stdlib/@tests/test_cases/builtins/check_slice.py

```diff
@@ -0,0 +1,107 @@
+from typing import Any
+from typing_extensions import assert_type
```

Comment:
(I would prefer for these imports to be moved below the docstring so that it is actually a docstring)